templates/template/README.md

```diff
@@ -207,7 +207,7 @@ No other changes are expected.
 
 {{- if (eq .Template "staging") }}
 
-`make prepare`: Pulls in your charts from upstream and creates a basic `generated-changes/` directory with your dependencies from upstream
+`make prepare`: Pulls in your charts from upstream and creates a basic `generated-changes/` directory with your dependencies from upstream. If you `export UPDATE_CRD_TEMPLATE=1`, it will automatically keep the CRD chart version up to date if the chart version bumps. You can also do this as a one-off action instead by running `./bin/charts-build-scripts prepare -u`.
```

Comment:
So when you have a single chart that you use a crd-template with, it will update the chart.yaml template ? I suppose its semi-helpful, but it adds more code when everything still works as expected beforehand. Has there been a problem with people not updating their CRD template or something ?

Reply:
Yeah Istio and CIS have had this issue before

Reply:
If you train a new SME its a new thing to miss though, wouldn't having the `make charts` command (or CI) validate this be better approach ?

Reply:
I agree with this. I'll close out this PR and update the issue description to validate that every CRD chart (that is specified via the package.yaml only; e.g. no validation will be done for something like [rancher-backup-crd](https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-backup-crd/rancher-backup-crd/1.0.400)) has a matching version with a main chart on `make charts` instead.